docs/general/vpn-issues.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'How to use AdGuard VPN in countries with Internet restrictions'
+sidebar_position: 10
+---
+
+In certain regions, such as China, Iran, and Russia, users might have problems connecting to AdGuard VPN. Here are the options that might help.
+
```

Comment:
First of all, you should explain how to install apps.

iOS: change store in App Store
Android: download apk from the website (mirror website)
Windows: download from the website
macOS: download from the website